subt/check_phone.py

```diff
@@ -0,0 +1,30 @@
+from osgar.logger import LogReader, lookup_stream_id
+from osgar.lib.serialize import deserialize
+
+CELL_PREFIX = "PhoneArtifact"
+
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser(description='Convert logfile to AVI video')
```

Comment:
The `description` is most likely forgotten copy & paste.

Reply:
Yes, fixed, thanks